doc/classes/Array.xml

```diff
@@ -291,7 +291,7 @@
 		</method>
 		<method name="shuffle">
 			<description>
-				Shuffles the array such that the items will have a random order.
+				Shuffles the array such that the items will have a random order. This function will generate reproducible values. To get random values in each runtime, consider using [method randomize].
```

Comment:
It should be `[method @GDScript.randomize]`. as it's a method described in the `@GDScript` documentation. `[method randomize]` would be valid if there was an `Array.randomize` method.

Could you amend the commit (`git commit --amend` and then `git push --force`) to update that?

Reply:
I'd also be less clear about the reproducibility of the values, as this method uses the shared RNG common to other global random methods (when not using a custom `RandomNumberGenerator`), so the shuffling you get is only reproducible if you always call random methods in the same order.

Something like this might be more accurate: "This method uses the global random number generator common to methods such as [method @GDScript.randi]. Call [method @GDScript.randomize] to ensure that a new seed will be used each time if you want non-reproducible shuffling."

Reply:
Hi, thanks for reviewing. Will make the appropriate changes now.